Client/metrics.yaml

```diff
@@ -760,6 +760,42 @@ firefox_home_page:
     notification_emails:
       - fx-ios-data-stewards@mozilla.com
     expires: "2022-05-01"
+  history_highlights_show_all:
+    type: counter
+    description: |
+      Counts the number of times a user taps the
+      Show All button in the History section
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-3470
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/9563
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    expires: "2022-06-01"
+  history_highlights_view:
+    type: counter
+    description: |
+      Counts the number of times a user sees the
+      History Highlights section on the homepage
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-3470
```

Comment:
We should only point to Github tickets (not JIRA) when it comes to data collections (we'll need to change the older ones that are not doing that), @nishant2718 told me that 😄

Reply:
The problem is that some tickets that exist is JIRA don't exist in GitHub. Currently, Stories in JIRA - which are meant for team only work that's not necessarily publicly planned - don't sync to GitHub; only Tasks do.

Reply:
Ahh I see! Then we can discuss that at some point I guess what to do in this case, since it was brought up [here](https://github.com/mozilla-mobile/firefox-ios/issues/9480). Marking this as resolved for now